event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepositoryIT.java

```diff
@@ -180,169 +180,6 @@ public void shouldThrowExceptionOnDuplicateSequenceId() throws InvalidSequenceId
         jdbcRepository.insert(eventOf(SEQUENCE_ID, STREAM_ID));
     }
 
-    @Test
```

Comment:
Are we still testing this?

Seems like we deleted quite a lot of tests.

Reply:
Yes testing for duplicate versions of event log